lib/stdio/type.js

```diff
@@ -29,7 +29,8 @@ export const isRegularUrl = stdioOption => isUrlInstance(stdioOption) && !hasFil
 
 const stringIsFilePath = stdioOption => stdioOption.startsWith('.') || isAbsolute(stdioOption);
 const isFilePath = stdioOption => typeof stdioOption === 'string' && stringIsFilePath(stdioOption);
-export const isUnknownStdioString = stdioOption => typeof stdioOption === 'string' && !stringIsFilePath(stdioOption) && !KNOWN_STDIO_STRINGS.has(stdioOption);
+
+export const isUnknownStdioString = (type, stdioOption) => type === 'native' && typeof stdioOption === 'string' && !KNOWN_STDIO_STRINGS.has(stdioOption);
```

Comment:
`native` type is when the `stdin`/`stdout`/`stderr` value is passed directly to `child_process.spawn()`, as opposed to being handled by Execa. In that case, only few specific strings are allowed.